types/office-js-preview/index.d.ts

```diff
@@ -984,9 +984,9 @@ declare namespace Office {
         /**
         * Check if the specified requirement set is supported by the host Office application.
         * @param name - Set name; e.g., "MatrixBindings".
-        * @param minVersion - The minimum required version; e.g., "1.4".
+        * @param minVersion - The minimum required version; e.g., "1.4". Note: Pass the minimum version parameter as a string data type to ensure you get the correct response. 
```

Comment:
Can we change this to be less vague about the consequences and more clear about our intentions?
"**Note**: Strings are the recommended parameter type. The use of number types is deprecated and will not be compatible with future requirement sets."

Reply:
Updated